reports/blueprint/report.tex

```diff
@@ -46,10 +46,29 @@
 
     \end{center}
 \end{titlepage}
-\section*{Executive Summary}
-% pop a one pager in here. Ideally less
-% Timothy
-\newpage
+\section{Executive Summary}
+
+This report outlines TeachEE, a device which facilitates online learning for
+electrical engineering labs. It describes the project's scope: a minimum viable
+product for the purpose of this capstone project, with room for scope expansion
+later in the year if circumstances allow it. The minimum viable product consists
+of a oscilloscope, desktop app, and communication between a computer and the
+device via USB. As a general rule, as long as waveforms can be produced in the
+hardware and displayed from software, a minimal product is satisfied.
+
+It also outlines a road-map for this project, and that we have so far completed
+most hardware related tasks from which we can now develop the software. Overall,
+it reports that the project is in a healthy state, showing progress in all
+domains, that all supplies will be delivered on time, and that the budget will
+be met. It identifies which members and responsible for what sections in the
+upcoming tasks, such that there is a clear and fair division of responsibility.
```

Comment:
End the sentence at upcoming tasks. And say "There is an equal division of responsibility between group members shown in Table x" use latex \ref directives.

Reply:
every single reference is broken for me and I can't be bothered to figure it out right now, so please check if the ref correctly renders for you

a7a58221674e9a7c15e4dc1acce40ef0bffa0c16